### PR TITLE
[TS] LPS-169550 Check objectActionExecutorKey value

### PR DESCRIPTION
**Pull request by TaiPhamLR**
Dear team,
This PR is for ticket https://issues.liferay.com/browse/LPS-169550 that was passed my review. Please help us to review it and leave your comments.
Thanks a lot.
cc @sontruongces

### CODE REVIEW

**Comment by sontruongces** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionBuilder.tsx [hunk 1]
```diff
@@ -599,7 +599,8 @@ export default function ActionBuilder({
 								if (value === 'add-object-entry') {
 									fetchObjectDefinitions();
 								}
-								setValues({
+
+								return values.objectActionExecutorKey === value ? '' : setValues({
 									objectActionExecutorKey: value,
 									parameters: {},
 								});
```
Comment (danvb112) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionBuilder.tsx]: Hey @TaiPhamLR and @sontruongces I think we can make this code simpler and more readable by inverting the condition, like:

```ts
if (values.objectActionExecutorKey !== value) {
    return setValues({
      objectActionExecutorKey: value,
      parameters: {},
    });
}
```

**Review by danvb112 — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionBuilder.tsx (hunk 1 above)
Comment (sontruongces) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionBuilder.tsx]: @danvb112, thank you for your feedback. The code was updated.

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by danvb112** [comment]
ci:forward

**Comment by naraandrade** [comment]
ci:forward

**Comment by danvb112** [comment]
Sent to: https://github.com/brianchandotcom/liferay-portal/pull/128035

**Pull request closed without merging**